src/main/java/com/faforever/client/map/MapVaultController.java

```diff
@@ -104,8 +95,16 @@ protected Node getEntityCard(MapBean map) {
 
   @Override
   protected List<ShowRoomCategory> getShowRoomCategories() {
```

Comment:
can u try test that? just have a look getRecommendedMapsWithPageCount is called with a valid page is called

Reply:
It doesn't actually call the function unless the showroom is loaded but I will see if I can mock it and test